articles/media-services/latest/live-event-error-codes.md

```diff
@@ -78,38 +75,34 @@ event.
 >|**MPE_RTMP_SESSION_IDLE_TIMEOUT** |
 >| Description|RTMP session timed out after being idle for allowed time limit. |
 >|Suggested solution|This typically happens when an encoder stops receiving the input feed so that the session becomes idle because there is no data to push out. Check if the encoder or input feed status is in a healthy state. |
->|**MPE_RTMP_FLV_TAG_TIMESTAMP_INVALID** |
->|Description| The timestamp for the video or audio FLVTag is invalid from RTMP encoder. |
->| Suggested solution| Deprecated. |
 >|**MPE_CAPACITY_LIMIT_REACHED** |
 >| Description|Encoder sending data too fast. |
->| Suggested solution|This happens when the encoder bursts out a large set of fragments in a brief period.  This can theoretically happen when the encoder can't push data for while due to a network issue and the bursts out data when the network is available. Find the reason from encoder log or system log. |
->|**Unknown error codes** |
->| Description| These error codes can range from memory error to duplicate entries in hash map. |
+>| Suggested solution|This happens when the encoder sends out a large set of fragments in a brief period.  This can theoretically happen when the encoder could not push data for while due to a network issue and then sends all the delayed fragments at once when the network becomes available. Find the reason please check the encoder logs. |
+
 
 ## Other error codes
 
 > [!div class="mx-tdCol2BreakAll"]
->| Error | Information |Rejected/Disconnected Event|
+>| Error | Information |Encoder disconnected/rejected|
 >|--|--|--|
 >|**ERROR_END_OF_MEDIA** ||Yes|
 >| Description|This is general error. ||
 >|Suggested solution| None.||
 >|**MPI_SYSTEM_MAINTENANCE** ||Yes|
 >| Description|The encoder disconnected due to service update or system maintenance. ||
->|Suggested solution|Make sure encoder enables 'auto connect'. This is encoder feature to recover the unexpected session disconnection. ||
+>|Suggested solution|Make sure the 'auto connect' is enabled for the contribution encoder. This allows the encoder to reconnect to the redundant live event endpoint that is not in maintenance. ||
```

Comment:
Acrolinx doesn't like the use of "This" without a noun after it. Change "This allows the encoder" to "It allows the encoder".  That way you have started with a noun.